routes/api_v1_0.py

```diff
@@ -802,10 +778,7 @@ def topo_v1_0(shaded_relief: str, projection: str, zoom: int, x: int, y: int):
         Generates topographical tiles
     """
 
-    if shaded_relief == "true":
-        bShaded_relief = True
-    else:
-        bShaded_relief = False
+    bShaded_relief = shaded_relief == "true"
```

Comment:
Nice

Reply:
Thanks, @htmlboss. I've been running this code in my dev container for the past couple of weeks. So, I'm confident enough in it to move it to staging for some other folks to exercise.

I agree that it would be nice to have a more uniform interface for opening sqlite, zarr, and dap. Will think on it...